src/auth/SignUp.jsx

```diff
@@ -64,8 +64,8 @@ export const SignUp = () => {
   return (
     <SignUpStyles>
       <GridAuth height='800px'>
-        <FormStlyes>
-          <div>
+        <FormStyles>
+          <div style={{ marginTop: 72 }}>
             <Title>
```

Comment:
We shouldn't use inline styling. it is a general rule of css that is good to follow.

Reply:
eliminated that.

Reply:
In general huge margins are not good to create space, I am also thinking about that now. It is better to use flex-box or grid to handle that, although some times is tricky.

Reply:
I know and understand. Please keep in mind the style its not responsive... . where are we starting and where are we stopping?

Reply:
we can fix all that, for sure, i just think we should be aware of that

Reply:
The centering is not about responsiveness it is about being in the center of the page, it is not related to be responsive. I am pretty sure we can avoid those margins to accomplish what we want, just letting grid or flexbox do its job.

Reply:
And again, it has nothing to do with responsiveness. Read about the uses of margins, it is quite rejected for css, I mean for handling centering of big spaces.

Reply:
I just put the 620px on the AvatarStyles that you put and the components are at the same level now.

Reply:
lets talk. the avatars JUST became a height - and are levelt. the other is for the sign up!

Reply:
Right, right. I mixed that my bad when writing.

Reply:
But still, we don't need that height for the Signup

Reply:
i did not set a height in signup. If i check in the browser, we need the 800px. why is the width 1600px?

Reply:
It is always good to put a max-with for big screens. 1048px and 1600px are popular in websites that do that.

Reply:
just asking, cause its not a max-width.